src/main/java/org/terasology/corerendering/rendering/dag/nodes/ShadowMapNode.java

```diff
@@ -162,18 +164,21 @@ public void process() {
             int numberOfChunksThatAreNotReadyYet = 0;
 
             final Vector3f cameraPosition = shadowMapCamera.getPosition();
+            shadowMapMaterial.setMatrix4("projection", shadowMapCamera.getProjectionMatrix(), true);
 
-            shadowMapCamera.lookThrough();
-
+            Matrix4f modelViewMatrix = new Matrix4f();
+            Matrix4f model = new Matrix4f();
```

Comment:
thought: JOML's whole zero-allocation mindset has got me noticing allocations in methods that get called every frame.

but at least this one is only once per frame, not in something that's going to get looped over a lot, and I promised myself I wouldn't do more premature optimization on rendering code :laughing: 

(after all, you're the one that's been spending the time looking at the profiler and memory allocations lately, not me)